css/ghost_host.css

```
@@ -0,0 +1,462 @@
+  color: rgb(255, 255, 255);
+}
+
+h1 {
+  font-size: 60px;
+  font-family: 'Lato';
+  text-transform: uppercase;
+  color: rgb(255, 255, 255);
+  margin-bottom: 25px;
+}
+
+.greetingHeader {
+  width: 500px;
+  /* padding-top: 340px; */
+  text-align: center;
+}
+
+.greetingHeader p {
+  font-size: 24px;
+  font-family: 'Lato';
+  color: rgb(255, 255, 255);
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
+.headerBottomHalf {
+  height: 120px;
+  width: 50%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.headerButton {
+  margin: auto;
+  height: 60px;
+  font-size: 20px;
+  font-family: 'Lato';
+  color: rgb(255, 255, 255);
+  background-color: transparent;
+  border-color: transparent;
+}
+
+.headerBottomHalf:first-child {
+  background-color: rgb(33, 66, 74)
+}
+
+.headerBottomHalf:nth-child(2) {
+  background-color: rgb(47, 172, 108)
+}
+
+.domainChoice {
+  height: 580px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  padding-top: 95px;
+  padding-bottom: 90px;
+}
+
+.domainChoice>div:first-child{
+  width: 1200px;
+  text-align: center;
+
+}
+
+.domainChoiceHead {
+  font-size: 20px;
+  font-family: 'Lato';
+  color: rgb(56, 55, 55);
+}
+
+.domainChoiceBody {
+  font-size: 20px;
+  font-family: 'Lato';
+  color: rgb(162, 162, 162);
+}
+
+.urlFirst {
+  width: 70px;
+  margin: auto;
+  color: rgb(205, 205, 205);
+  font-size: 23px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.urlSecond {
+  width: 730px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.urlThird {
+  width: 85px;
+  color: rgb(205, 205, 205);
+  font-size: 23px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+
+}
+
+.domainChoicePart{
+  height: 50px;
+  border: 1px solid rgb(162, 162, 162);
+  border-left: 0.5px;
+}
+
+.domainChoiceRow {
+  width: 890px;
+  border-left: 1px solid rgb(162, 162, 162);
+  font-family: 'Lato';
+}
+
+.secondLevelDomain {
+  width: 730px;
+  font-size: 23px;
+  color: rgb(134, 134, 134);
+  border: 0cm;
+  font-family: 'Lato';
+}
+
+.firstLevelDomain {
+  color: rgb(205, 205, 205);
+  font-size: 23px;
+  border: 0cm;
+}
+
+.planCompareTable{
+  margin: 0 auto;
+  width: 890px;
+  display: flex;
+  flex-direction: column;
+  height: 194px;
+  font-size: 13px;
+  font-family: 'Lato';
+  color: rgb(224, 224, 224);
+  border: 1px solid rgb(162, 162, 162);
+}
+
+.planCompareTableSection {
+  height: 98px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.packageImage{
+  width: 15px;
+  height: 15px;
+}
+
+.packageSection {
+  background: url(../img/package_scenery.png) no-repeat;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  height: 765px;
+}
+
+.packageDiv{
+  padding-top: 100px;
+  width: 1200px;
+}
+
+.packageDiv div p {
```

better to write more specified selectors, div here is not a best solution 
```suggestion
.packageDiv .some-class-name p {
```